### PR TITLE
[RHELC-1067] Automatically tag every Friday

### PR DESCRIPTION
**Pull request by SpyTec**
Jira Issues: [RHELC-1067](https://issues.redhat.com/browse/RHELC-1067)

Checklist

- [ ] PR has been tested manually in a VM (either author or reviewer)
- [ ] Jira issue has been made public if possible
- [x] `[RHELC-]` is part of the PR title 
- [ ] GitHub label has been added to help with Release notes 
- [ ] PR title explains the change from the user's point of view
- [ ] Code and tests are documented properly
- [ ] The commits are squashed to as few commits as possible (without losing data) 
- [ ] When merged: Jira issue has been updated to `Release Pending` if relevant

### CODE REVIEW

File: .github/workflows/autotag.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: Auto Tag
+
+on:
+  schedule:
+    - cron: "1 12 * * 5"
```
Comment (r0x0d) [inline .github/workflows/autotag.yml:5]: Looks good to me, but why the `1` there instead of `0`? Any particular reason?

**Review by r0x0d — APPROVED** [review]

File: .github/workflows/autotag.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        with:
+          fetch-depth: 0
+
+      - name: Get Latest Tag
+        id: current_tag
+        run: echo "tag=$(git describe --tags --abbrev=0)" >> $GITHUB_OUTPUT
+
+      - name: Create tag
+        id: create_tag
+        run: echo "tag=${{ steps.current_tag.outputs.tag }}-${{ github.run_number }}" >> $GITHUB_OUTPUT
```
Comment (r0x0d) [inline .github/workflows/autotag.yml:23]: Maybe we could simplify? 

```suggestion
      - name: Create tag
        id: create_tag
        run: echo "tag=$(git describe --tags --abbrev=0)-${{ github.run_number }}" >> $GITHUB_OUTPUT
```

**Comment by r0x0d** [comment]
Also, probably you have come across this already, but found out this action while searching for alternatives: https://github.com/actions-ecosystem/action-push-tag. 

It seems quite simple and efficient.

File: .github/workflows/autotag.yml (hunk 1 above)
Comment (SpyTec) [reply .github/workflows/autotag.yml:5]: I always went into the habit of not doing round numbers for scheduled stuff due to everyone doing so. So there are usually spikes of activity around the top of the hour on CI infra where you can do cronjobs

File: .github/workflows/autotag.yml (hunk 2 above)
Comment (SpyTec) [reply .github/workflows/autotag.yml:23]: True, doesn't serve much purpose to split it

**Comment by SpyTec** [comment]
> Also, probably you have come across this already, but found out this action while searching for alternatives: [actions-ecosystem/action-push-tag](https://github.com/actions-ecosystem/action-push-tag).
> 
> It seems quite simple and efficient.

@r0x0d I did see it but felt like it didn't give any benefit to doing it manually. Especially since it adds a dependency that does something super simple

File: .github/workflows/autotag.yml (hunk 1 above)
Comment (r0x0d) [reply .github/workflows/autotag.yml:5]: Sounds good! Thanks.

**Pull request merged**